src/outbox/outboxReducers.js

```diff
@@ -9,6 +9,7 @@ const reducer = (state: OutboxState = initialState, action: Action): OutboxState
     case MESSAGE_SEND: {
       return [...state, { ...action.params }];
     }
+    case DELETE_OUTBOX_MESSAGE:
```

Comment:
When a new message comes in we need to check if the id matches then delete from reducer and similar thing is done in the case of `DELETE_OUTBOX_MESSAGE` hence we can have the same block!